src/App.tsx

```diff
@@ -1,48 +1,50 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+import {
+  Navigate,
+  NavLink,
+  Route,
+  Routes,
+} from 'react-router-dom';
+import { Tabs } from './components/Tabs/Tabs';
+import { Home } from './components/Home/Home';
+import { NotFoundPage } from './components/NotFoundPage/NotFoundPage';
 
-// const tabs = [
-//   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
-//   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
-//   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
-// ];
+type ActiveLink = (status: { isActive: boolean }) => string;
 
-const App = () => (
-  <>
-    {/* Also requires <html class="has-navbar-fixed-top"> */}
-    <nav className="navbar is-fixed-top has-background-light" data-cy="nav">
-      <div className="navbar-menu">
-        <div className="navbar-start">
-          <a href="/" className="navbar-item isActive">Home</a>
-          <a href="/tabs" className="navbar-item isActive">Tabs</a>
-        </div>
-      </div>
-    </nav>
+const tabs = [
+  { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
+  { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
+  { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
+];
 
-    <div className="section">
-      <h1 className="title">Home page</h1>
-      <h1 className="title">Tabs page</h1>
-      <h1 className="title">Page not found</h1>
+const App: React.FC = () => {
+  const linkStatus: ActiveLink = ({ isActive }) => (
+    isActive ? 'navbar-item is-active' : 'navbar-item');
```

Comment:
```suggestion
  const linkStatus: ActiveLink = ({ isActive }) => (
    isActive 
      ? 'navbar-item is-active' 
      : 'navbar-item'
  );
```

Reply:
however, as you can see, you duplicate yourself, so it would be great to apply classnames here
```
className={({ isActive }) => classNames(
  'navbar-item',
  { isActive },
)}
```
here is a simple solution